build/generate-style-spec.ts

```diff
@@ -315,6 +317,10 @@ export type DataDrivenPropertyValueSpecification<T> =
     | CompositeFunctionSpecification<T>
     | ExpressionSpecification;
 
+
+export type StateSpecification = Record<string, SchemaSpecification>;
+export type SchemaSpecification = Record<string, unknown>;
```

Comment:
I think this can be improved if there's a schema definition for type, default etc, can't it?

Reply:
Done